README.md

```diff
@@ -176,10 +180,14 @@ Documentation
 ```js
 const drivelist = require('drivelist');
 
-const drives = await drivelist.list();
-drives.forEach((drive) => {
-  console.log(drive);
-});
+getDrives();
+
+async function getDrives () {
+    const drives = await drivelist.list();
+    drives.forEach((drive) => {
+        console.log(drive);
+      });
```

Comment:
Please fix the indentation.
You also need to add a footer to your commit message:
```

Change-type: patch
```

Reply:
It is not a failure. You just have to add async to the function callback ...